### ISSUE TITLE
Rule proposal: sorted props

### ISSUE THREAD

**Issue by lencioni** [yannickcr/eslint-plugin-react#16]
Some people might want to enforce that their props are sorted alphabetically.

Good:

```jsx
<MyComponent
  aProp="value"
  bProp="value"
  cProp="value"
/>
```

Bad:

```jsx
<MyComponent
  cProp="value"
  aProp="value"
  bProp="value"
/>
```
